### PR TITLE
Bug/approve bnb

### PR DESCRIPTION
**Pull request by fakoredeDamilola**
This removes the approve button for bnb

### CODE REVIEW

**Comment by fakoredeDamilola** [comment]
https://www.loom.com/share/69c8b19fed4d41c6913913373fed8364

File: app/components/liquidity/from.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const LiquidityFromBox = ({
   setFromInputMax,
   label,
 }) => {
+  console.log({fromSelectedToken})
```
Comment (kayroy247) [inline app/components/liquidity/from.js:22]: Kindly remove the console log statements on your PR

File: app/containers/LiquidityPage/index.js [hunk 2]
```diff
@@ -182,14 +182,15 @@ export function LiquidityPage(props) {
     getBalance()
   }, [percentValue, wallet])
 
-  const handleFromAmount = () => {
+  const handleFromAmount = (liquidityRatio = liquidityPairRatio) => {
     if(!newTokenPairButton){
-      setToValue((fromValue * liquidityPairRatio).toString());
+      console.log({fromValue,liquidityRatio})
```
Comment (kayroy247) [inline app/containers/LiquidityPage/index.js:187]: Kindly remove the above console log statement

File: app/containers/LiquidityPage/index.js [hunk 3]
```diff
@@ -244,13 +245,24 @@ if (LPAddress !== "0x0000000000000000000000000000000000000000" ){
...
+      console.log(liquidityRatio)
+      getWhichValueHasChanged(liquidityRatio)
     } catch (error) {
       console.error(error)
     }
 
   }
+
+const getWhichValueHasChanged = (liquidityRatio) =>{
+  console.log('GETTING')
```
Comment (kayroy247) [inline app/containers/LiquidityPage/index.js:257]: Kindly remove the console log statements

File: app/containers/LiquidityPage/index.js [hunk 4]
```diff
@@ -244,13 +245,24 @@ if (LPAddress !== "0x0000000000000000000000000000000000000000" ){
         liquidityRatio = tokenAReserve.toString() / tokenBreserve.toString();
       }
       setLiquidityPairRatio(liquidityRatio);
+      console.log(liquidityRatio)
```
Comment (kayroy247) [inline app/containers/LiquidityPage/index.js:248]: Kindly remove this console log statement also.

**Review by kayroy247 — COMMENTED** [review]
Kindly implement the feedback.

**Comment by kayroy247** [comment]
Also remove the package-lock.json file. Our package management library is yarn so we don't expect to see package-lock.json on the codebase but yarn-lock.json

**Pull request merged**